recipe/meta.yaml

```diff
@@ -1,35 +1,88 @@
-{% set name = "openfst" %}
 {% set version = "1.8.2" %}
 
 package:
-  name: {{ name|lower }}
+  name: openfst
```

Comment:
Please revert unless you have some updated style information. This is what all the formulae did when I wrote this.

Reply:
The default formula is to make the name a variable so that things like the pypi link can be auto-generated, but it is fake variability (the variable can never be changed), and it should be changed. I do not intend to change this.

Reply:
+1